jsondoc-ui-webjar/src/main/resources/jsondoc-ui.html

```diff
@@ -1016,6 +1040,14 @@ <h5 style="padding:0px">Request URL</p>
 							if(method.auth.type == "BASIC_AUTH") {
 								headers["Authorization"] = "Basic " + window.btoa($('#basicAuthUsername').val() + ":" + $('#basicAuthPassword').val());
 							}
+							
+							if(method.auth.type == "TOKEN") {
+								if(method.auth.scheme != "") {
```

Comment:
If scheme is set to "" servlet doesn't return the field. Need it fix this line with:
```
if(typeof(method.auth.scheme) != "undefined" && method.auth.scheme != "") {
```